hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip991/TopicCustomFeeSubmitMessageTest.java

```diff
@@ -410,5 +413,96 @@ final Stream<DynamicTest> collectorSubmitMessageToTopicWith2differentFees() {
                     getAccountBalance(collector).hasTokenBalance(BASE_TOKEN, 0L),
                     getAccountBalance(secondCollector).hasTokenBalance(SECOND_TOKEN, 1L));
         }
+
+        @HapiTest
+        // TOPIC_FEE_177
+        @DisplayName("Submits messages with account excluded and included from FEKL")
+        final Stream<DynamicTest> submitWithoutAndWittFEKL() {
```

Comment:
submitWithoutAndWittFEKL to submitWithoutAndWithFEKL

Reply:
Fixed